HousingRegisterApi/V1/Gateways/DynamoDbGateway.cs

```diff
@@ -67,6 +68,23 @@ public Application UpdateApplication(Guid id, UpdateApplicationRequest request)
                 entity.OtherMembers = request.OtherMembers.ToList();
 
             _dynamoDbContext.SaveAsync(entity).GetAwaiter().GetResult();
+
+            return entity.ToDomain();
+        }
+
+        public Application CompleteApplication(Guid id)
+        {
+            var entity = _dynamoDbContext.LoadAsync<ApplicationDbEntity>(id).GetAwaiter().GetResult();
+            if (entity == null)
+            {
+                return null;
+            }
+
+            entity.SubmittedAt = DateTime.UtcNow;
```

Comment:
We should apply some checks here to ensure we have something that looks like a valid application to complete, otherwise we may as well just reuse the `UpdateApplication` endpoint and pass in the values within the request.

Reply:
I've checked to see if the main applicant is present before allowing the application to be set as completed